### PR TITLE
feat: support v2 bCake farm on ARB

### PR DESCRIPTION
**Pull request by Chef-Yogi**
---

## PR-Codex overview
This PR introduces support for `ChainId.ARBITRUM_ONE`, adds `bCakeWrapperAddress` to farm configs, and updates UI elements based on `ChainId.BSC`.

### Detailed summary
- Added support for `ChainId.ARBITRUM_ONE`
- Included `bCakeWrapperAddress` in farm configurations
- Updated UI elements based on `ChainId.BSC` for better user experience

> The following files were skipped due to too many changes: `apps/web/src/views/Farms/components/FarmCard/FarmCard.tsx`, `apps/web/src/state/farms/index.ts`, `packages/farms/constants/arb.ts`, `packages/widgets-internal/farm/components/FarmCard/DetailsSection.tsx`, `apps/web/src/views/Farms/components/FarmTable/Row.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/farms/src/v2/fetchFarmsV2.ts [hunk 1]
```diff
@@ -132,6 +137,8 @@ export async function farmV2FetchFarms({
     decimals,
   )
 
+  console.log(farmsDataWithPrices, 'farmsDataWithPrices????')
```
[the remark this answers is not in the document]
Comment (Chef-Yogi) [reply packages/farms/src/v2/fetchFarmsV2.ts:140]: sure

File: apps/web/src/views/Farms/components/FarmCard/FarmCard.tsx [hunk 2]
```diff
@@ -62,6 +62,7 @@ const FarmCard: React.FC<React.PropsWithChildren<FarmCardProps>> = ({
   account,
   originalLiquidity,
 }) => {
+  // console.log(farm, 'farmCard????')
```
[the remark this answers is not in the document]
Comment (Chef-Yogi) [reply apps/web/src/views/Farms/components/FarmCard/FarmCard.tsx:65]: thanks I will remove this later, still observe the data from multichain

File: packages/farms/constants/arb.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { arbitrumTokens } from '@pancakeswap/tokens'
 import { FeeAmount } from '@pancakeswap/v3-sdk'
-import { FarmConfigV3 } from '../src'
+import { getAddress } from 'viem'
+import { FarmConfigV3, SerializedFarmConfig } from '../src'
```
Comment (chefjackson) [inline packages/farms/constants/arb.ts:4]: pls import directly from types

Comment (Chef-Yogi) [reply packages/farms/constants/arb.ts]: import { FarmConfigV3, SerializedFarmConfig } from '../src/types'
like this ?

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**